packages/core/src/card/Card.css

```diff
@@ -30,7 +30,7 @@
 a:focus .uitkCard-interactable,
 .uitkCard-interactable:hover {
   border-color: var(--uitkCard-borderColor-hover, var(--uitk-selectable-borderColor-hover));
-  color: var(--uitkCard-color-hover, var(--uitk-text-primary-foreground));
+  color: var(--uitkCard-color-disabled, var(--uitk-text-primary-foreground-hover));
```

Comment:
Should this be disabled?

Reply:
oh yes i fixed that in another one and then this has the old in still, will delete